File: 1205-defanging-an-ip-address/defanging-an-ip-address.cpp
class Solution {
public:
    string defangIPaddr(string address) {
        string p;
        int n = address.size();
        int j = 0;
        for (int i = 0; i < n; i++) {
            if (address[i] == '.') {
                p += "[.]";
            } else {
                p += address[i];
            }
        }
        return p;
    }
};

File: 1205-defanging-an-ip-address/defanging-an-ip-address.java
class Solution {
    public String defangIPaddr(String address) {
        String p = "";
        int n = address.length();
        for (int i = 0; i < n; i++) {
            if (address.charAt(i) == '.') {
                p += "[.]";
            } else {
                p += address.charAt(i);
            }
        }
        return p;
    }
}